docs/abac-rocksdb-label-store.md

```diff
@@ -235,4 +239,6 @@ This did indeed reduce the efficiency of the compression:
 * Open telemetry
 * Stack trace, log, debug
 * Making label store queryable - for investigation
-* Performance testing - various Store Formats (at higher level) 
\ No newline at end of file
+* Performance testing - various Store Formats (at higher level) 
+* Starting a LabelStore with the wrong storage format
```

Comment:
Not for this PR but should probably have a Backlog ticket for writing some sort of marker file so we can compare what the user has configured with what the Labels Store was previously written as and bail if there's a mismatch

Even longer term it might be nice to support migrating between some formats e.g. converting an existing store from By String to By Hash, again separate backlog ticket

Reply:
Had similar thoughts. Articulating them in tickets is the trickier part.